### PR TITLE
fix: Query conditions `inQuery` and `notInQuery` not working properly

### PR DESCRIPTION
**Pull request by mbfakourii**
### New Pull Request Checklist
- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).

### Issue Description
There was an unused "," value in the _buildQueryRelational function

Closes: #849

### Approach
n/a

### TODOs before merging
- [ ] Add tests
- [ ] Add changes to documentation (guides, repository pages, in-code descriptions)
- [ ] A changelog entry

### CODE REVIEW

**Comment by mtrezza** [comment]
Is this ready for review?

**Comment by mbfakourii** [comment]
> Is this ready for review?

Yes, it is ready for review

**Review by Nidal-Bakir — APPROVED** [review]

**Comment by Nidal-Bakir** [comment]
Can you add a test to ensure that this does not happen again?

**Comment by mbfakourii** [comment]
> Can you add a test to ensure that this does not happen again?

Yes, sure

**Comment by mbfakourii** [comment]
@mtrezza 
Why did half of the CI failed?

**Comment by Nidal-Bakir** [comment]
> Why did half of the CI failed?

See https://github.com/parse-community/Parse-SDK-Flutter/pull/874.

**Review by Nidal-Bakir — APPROVED** [review]

File: packages/dart/test/src/network/parse_query_test.dart [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import 'dart:convert';
```
Comment (Nidal-Bakir) [inline packages/dart/test/src/network/parse_query_test.dart:1]: `import 'dart:convert'; `

Why was this removed?

**Review by Nidal-Bakir — CHANGES_REQUESTED** [review]

File: packages/dart/test/src/network/parse_query_test.dart (hunk 1 above)
Comment (mbfakourii) [reply packages/dart/test/src/network/parse_query_test.dart:1]: I think it was deleted during the merge. Thank you for saying that I will fix it

**Review by Nidal-Bakir — APPROVED** [review]
Thanks

**Review by mtrezza — APPROVED** [review]
Looks good, could you just rephrase the PR title, it seems to describe the technical solution instead of the bug that is fixed by this PR; so that we have a meaningful changelog entry

**Comment by mbfakourii** [comment]
> Looks good, could you just rephrase the PR title, it seems to describe the technical solution instead of the bug that is fixed by this PR; so that we have a meaningful changelog entry

I changed
The problem is very small. I don't have a better idea for the title

**Review by mtrezza — CHANGES_REQUESTED** [review]
Could you add a changelog entry?

**Comment by mbfakourii** [comment]
> Could you add a changelog entry?

added

**Review by mtrezza — APPROVED** [review]
Looks good! Waiting for CI...
